### PR TITLE
[BEAM-12572] Java examples should get continuously exercised on at least 2 runners

### PR DESCRIPTION
**Pull request by benWize**
**Please** add a meaningful description for your change here

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] [**Choose reviewer(s)**](https://beam.apache.org/contribute/#make-your-change) and mention them in a comment (`R: @username`).
 - [ ] Format the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`, where you replace `BEAM-XXX` with the appropriate JIRA issue, if applicable. This will automatically link the pull request to the issue.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/#make-reviewers-job-easier).

`ValidatesRunner` compliance status (on master branch)
--------------------------------------------------------

<table>
  <thead>
    <tr>
      <th>Lang</th>
      <th>ULR</th>
      <th>Dataflow</th>
      <th>Flink</th>
      <th>Samza</th>
      <th>Spark</th>
      <th>Twister2</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Go</td>
      <td>---</td>
      <td>
        <a href="https://ci-beam.apache.org/job/beam_PostCommit_Go/lastCompletedBuild/">
          Build Status
        </a>
      <td>
        <a href="https://ci-beam.apache.org/job/beam_PostCommit_Go_VR_Flink/lastCompletedBuild/">
          Build Status
        </a>
      </td>
      <td>
        <a href="https://ci-beam.apache.org/job/beam_PostCommit_Go_VR_Samza/lastCompletedBuild/">
          Build Status
        </a>
      </td>
      <td>
        <a href="https://ci-beam.apache.org/job/beam_PostCommit_Go_VR_Spark/lastCompletedBuild/">
          Build Status
        </a>
      </td>
      <td>---</td>
    </tr>
    <tr>
      <td>Java</td>
      <td>
        <a href="https://ci-beam.apache.org/job/beam_PostCommit_Java_ValidatesRunner_ULR/lastCompletedBuild/">
          Build Status
        </a>
      </td>
      <td>
        <a href="https://ci-beam.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Dataflow/lastCompletedBuild/">
          Build Status
        </a><br>
        <a href="https://ci-beam.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Dataflow_Streaming/lastCompletedBuild/">
          Build Status
        </a><br>
        <a href="https://ci-beam.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Dataflow_Java11/lastCompletedBuild/">
          Build Status
        </a><br>
        <a href="https://ci-beam.apache.org/job/beam_PostCommit_Java_VR_Dataflow_V2/lastCompletedBuild/">
          Build Status
        </a><br>
        <a href="https://ci-beam.apache.org/job/beam_PostCommit_Java_VR_Dataflow_V2_Streaming/lastCompletedBuild/">
          Build Status
        </a><br>
      </td>
      <td>
        <a href="https://ci-beam.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Flink/lastCompletedBuild/">
          Build Status
        </a><br>
        <a href="https://ci-beam.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Flink_Java11/lastCompletedBuild/">
          Build Status
        </a><br>
        <a href="https://ci-beam.apache.org/job/beam_PostCommit_Java_PVR_Flink_Batch/lastCompletedBuild/">
          Build Status
        </a><br>
        <a href="https://ci-beam.apache.org/job/beam_PostCommit_Java_PVR_Flink_Streaming/lastCompletedBuild/">
          Build Status
        </a>
      </td>
      <td>
        <a href="https://ci-beam.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Samza/lastCompletedBuild/">
          Build Status
        </a><br>
        <a href="https://ci-beam.apache.org/job/beam_PostCommit_Java_PVR_Samza/lastCompletedBuild/">
          Build Status
        </a>
      </td>
      <td>
        <a href="https://ci-beam.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Spark/lastCompletedBuild/">
          Build Status
        </a><br>
        <a href="https://ci-beam.apache.org/job/beam_PostCommit_Java_PVR_Spark_Batch/lastCompletedBuild/">
          Build Status
        </a><br>
        <a href="https://ci-beam.apache.org/job/beam_PostCommit_Java_ValidatesRunner_SparkStructuredStreaming/lastCompletedBuild/">
          Build Status
        </a>
      </td>
      <td>
        <a href="https://ci-beam.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Twister2/lastCompletedBuild/">
          Build Status
        </a>
      </td>
    </tr>
    <tr>
      <td>Python</td>
      <td>---</td>
      <td>
        <a href="https://ci-beam.apache.org/job/beam_PostCommit_Py_VR_Dataflow/lastCompletedBuild/">
          Build Status
        </a><br>
        <a href="https://ci-beam.apache.org/job/beam_PostCommit_Py_VR_Dataflow_V2/lastCompletedBuild/">
          Build Status
        </a><br>
        <a href="https://ci-beam.apache.org/job/beam_PostCommit_Py_ValCont/lastCompletedBuild/">
          Build Status
        </a>
      </td>
      <td>
        <a href="https://ci-beam.apache.org/job/beam_PreCommit_Python_PVR_Flink_Cron/lastCompletedBuild/">
          Build Status
        </a><br>
        <a href="https://ci-beam.apache.org/job/beam_PostCommit_Python_VR_Flink/lastCompletedBuild/">
          Build Status
        </a>
      </td>
      <td>
        <a href="https://ci-beam.apache.org/job/beam_PostCommit_Python_VR_Samza/lastCompletedBuild/">
          Build Status
        </a>
      </td>
      <td>
        <a href="https://ci-beam.apache.org/job/beam_PostCommit_Python_VR_Spark/lastCompletedBuild/">
          Build Status
        </a>
      </td>
      <td>---</td>
    </tr>
    <tr>
      <td>XLang</td>
      <td>
        <a href="https://ci-beam.apache.org/job/beam_PostCommit_XVR_Direct/lastCompletedBuild/">
          Build Status
        </a>
      </td>
      <td>
        <a href="https://ci-beam.apache.org/job/beam_PostCommit_XVR_Dataflow/lastCompletedBuild/">
          Build Status
        </a>
      </td>
      <td>
        <a href="https://ci-beam.apache.org/job/beam_PostCommit_XVR_Flink/lastCompletedBuild/">
          Build Status
        </a>
      </td>
      <td>
        <a href="https://ci-beam.apache.org/job/beam_PostCommit_XVR_Samza/lastCompletedBuild/">
          Build Status
        </a>
      </td>
      <td>
        <a href="https://ci-beam.apache.org/job/beam_PostCommit_XVR_Spark/lastCompletedBuild/">
          Build Status
        </a>
      </td>
      <td>---</td>
    </tr>
  </tbody>
</table>

Examples testing status on various runners
--------------------------------------------------------

<table>
  <thead>
    <tr>
      <th>Lang</th>
      <th>ULR</th>
      <th>Dataflow</th>
      <th>Flink</th>
      <th>Samza</th>
      <th>Spark</th>
      <th>Twister2</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Go</td>
      <td>---</td>
      <td>---</td>
      <td>---</td>
      <td>---</td>
      <td>---</td>
      <td>---</td>
      <td>---</td>
    </tr>
    <tr>
      <td>Java</td>
      <td>---</td>
      <td>
        <a href="https://ci-beam.apache.org/job/beam_PreCommit_Java_Examples_Dataflow_Cron/lastCompletedBuild/">
          Build Status
        </a><br>
        <a href="https://ci-beam.apache.org/job/beam_PreCommit_Java_Examples_Dataflow_Java11_Cron/lastCompletedBuild/">
          Build Status
        </a><br>
        <a href="https://ci-beam.apache.org/job/beam_PostCommit_Java_Examples_Dataflow_V2/lastCompletedBuild/">
          Build Status
        </a><br>
      </td>
      <td>---</td>
      <td>---</td>
      <td>---</td>
      <td>---</td>
      <td>---</td>
    </tr>
    <tr>
      <td>Python</td>
      <td>---</td>
      <td>---</td>
      <td>---</td>
      <td>---</td>
      <td>---</td>
      <td>---</td>
      <td>---</td>
    </tr>
    <tr>
      <td>XLang</td>
      <td>---</td>
      <td>---</td>
      <td>---</td>
      <td>---</td>
      <td>---</td>
      <td>---</td>
      <td>---</td>
    </tr>
  </tbody>
</table>

Post-Commit SDK/Transform Integration Tests Status (on master branch)
------------------------------------------------------------------------------------------------

<table>
  <thead>
    <tr>
      <th>Go</th>
      <th>Java</th>
      <th>Python</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>
        <a href="https://ci-beam.apache.org/job/beam_PostCommit_Go/lastCompletedBuild/">
          Build Status
        </a>
      </td>
      <td>
        <a href="https://ci-beam.apache.org/job/beam_PostCommit_Java/lastCompletedBuild/">
          Build Status
        </a>
      </td>
      <td>
        <a href="https://ci-beam.apache.org/job/beam_PostCommit_Python36/lastCompletedBuild/">
          Build Status
        </a><br>
        <a href="https://ci-beam.apache.org/job/beam_PostCommit_Python37/lastCompletedBuild/">
          Build Status
        </a><br>
        <a href="https://ci-beam.apache.org/job/beam_PostCommit_Python38/lastCompletedBuild/">
          Build Status
        </a>
      </td>
    </tr>
  </tbody>
</table>

Pre-Commit Tests Status (on master branch)
------------------------------------------------------------------------------------------------

<table>
  <thead>
    <tr>
      <th>---</th>
      <th>Java</th>
      <th>Python</th>
      <th>Go</th>
      <th>Website</th>
      <th>Whitespace</th>
      <th>Typescript</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Non-portable</td>
      <td>
        <a href="https://ci-beam.apache.org/job/beam_PreCommit_Java_Cron/lastCompletedBuild/">
          Build Status
        </a><br>
      </td>
      <td>
        <a href="https://ci-beam.apache.org/job/beam_PreCommit_Python_Cron/lastCompletedBuild/">
          Build Status
        </a><br>
        <a href="https://ci-beam.apache.org/job/beam_PreCommit_PythonLint_Cron/lastCompletedBuild/">
          Build Status
        </a><br>
        <a href="https://ci-beam.apache.org/job/beam_PreCommit_PythonDocker_Cron/lastCompletedBuild/">
          Build Status
        </a><br>
        <a href="https://ci-beam.apache.org/job/beam_PreCommit_PythonDocs_Cron/lastCompletedBuild/">
          Build Status
        </a>
      </td>
      <td>
        <a href="https://ci-beam.apache.org/job/beam_PreCommit_Go_Cron/lastCompletedBuild/">
          Build Status
        </a>
      </td>
      <td>
        <a href="https://ci-beam.apache.org/job/beam_PreCommit_Website_Cron/lastCompletedBuild/">
          Build Status
        </a>
      </td>
      <td>
        <a href="https://ci-beam.apache.org/job/beam_PreCommit_Whitespace_Cron/lastCompletedBuild/">
          Build Status
        </a>
      </td>
      <td>
        <a href="https://ci-beam.apache.org/job/beam_PreCommit_Typescript_Cron/lastCompletedBuild/">
          Build Status
        </a>
      </td>
    </tr>
    <tr>
      <td>Portable</td>
      <td>---</td>
      <td>
        <a href="https://ci-beam.apache.org/job/beam_PreCommit_Portable_Python_Cron/lastCompletedBuild/">
          Build Status
        </a>
      </td>
      <td>
        <a href="https://ci-beam.apache.org/job/beam_PreCommit_GoPortable_Cron/lastCompletedBuild/">
          Build Status
        </a>
      </td>
      <td>---</td>
      <td>---</td>
      <td>---</td>
    </tr>
  </tbody>
</table>

See [.test-infra/jenkins/README](https://github.com/apache/beam/blob/master/.test-infra/jenkins/README.md) for trigger phrase, status and link of all Jenkins jobs.

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI.

### CODE REVIEW

**Comment by benWize** [comment]
Run Java Examples_Direct

**Comment by benWize** [comment]
Run Java Examples_Dataflow

**Comment by benWize** [comment]
Run Java Examples_Flink

**Comment by benWize** [comment]
Run Java Examples_Spark

**Comment by benWize** [comment]
Run Java Examples_Direct

**Comment by benWize** [comment]
Run Java Examples_Flink

**Comment by benWize** [comment]
Run Java Examples_Spark

**Comment by benWize** [comment]
Run Java Examples on Dataflow Runner V2

**Comment by benWize** [comment]
Run Java Examples_Direct

**Comment by benWize** [comment]
Run Java Examples on Dataflow Runner V2

**Comment by benWize** [comment]
Run Java Examples_Flink

**Comment by benWize** [comment]
Run Java Examples_Spark

**Comment by benWize** [comment]
Run Java Examples_Direct

**Comment by benWize** [comment]
Run Java Examples_Direct

**Comment by benWize** [comment]
Run Java Examples_Spark

**Comment by benWize** [comment]
Run Java Examples_Flink

**Comment by benWize** [comment]
Run Java Examples on Dataflow Runner V2

**Comment by benWize** [comment]
Run Java Examples on Dataflow Runner V2

**Comment by benWize** [comment]
@tvalentyn could you help me to review this? I added some tests to check examples end to end. There are some test examples still in progress and some others are disabled because they failed for some runners, but it could be useful to have an early review.

**Comment by benWize** [comment]
Tests executed in each runner:

- [Direct](https://ci-beam.apache.org/job/beam_PostCommit_Java_Examples_Direct/lastCompletedBuild/testReport/)
- [Spark](https://ci-beam.apache.org/job/beam_PostCommit_Java_Examples_Spark/4/testReport/)
- [Flink](https://ci-beam.apache.org/job/beam_PostCommit_Java_Examples_Flink/4/testReport/)
- [Dataflow](https://ci-beam.apache.org/job/beam_PostCommit_Java_Examples_Dataflow_V2/1784/testReport/)

File: examples/java/src/test/java/org/apache/beam/examples/cookbook/MaxPerKeyExamplesIT.java [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    this.bqClient.deleteDataset(this.projectId, this.outputDatasetId);
+  }
+
+  @Test
+  public void testE2EMaxPerKeyExamples() throws Exception {
+    this.options.setOutput(
+        String.format("%s:%s.%s", this.projectId, this.outputDatasetId, this.outputTable));
+    MaxPerKeyExamples.runMaxPerKeyExamples(options);
+    FluentBackoff backoffFactory =
+        FluentBackoff.DEFAULT.withMaxRetries(4).withInitialBackoff(Duration.standardSeconds(1L));
```
Comment (tvalentyn) [inline examples/java/src/test/java/org/apache/beam/examples/cookbook/MaxPerKeyExamplesIT.java:77]: how long is the longest back-off period?

**Comment by tvalentyn** [comment]
took a quick look, but will defer to @kileys for final review&merge.

File: examples/java/src/test/java/org/apache/beam/examples/cookbook/MaxPerKeyExamplesIT.java (hunk 1 above)
Comment (benWize) [reply examples/java/src/test/java/org/apache/beam/examples/cookbook/MaxPerKeyExamplesIT.java:77]: I think is like 2 minutes max, the default exponent is 1.5 and the max retries are 4.

File: examples/java/src/test/java/org/apache/beam/examples/cookbook/CombinePerKeyExamplesIT.java [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+  public interface CombinePerKeyExamplesOptions
+      extends TestPipelineOptions, CombinePerKeyExamples.Options {}
+
+  @Before
+  public void setupTestEnvironment() throws Exception {
+    PipelineOptionsFactory.register(TestPipelineOptions.class);
+    this.options = TestPipeline.testingPipelineOptions().as(CombinePerKeyExamplesOptions.class);
+    this.projectId = TestPipeline.testingPipelineOptions().as(GcpOptions.class).getProject();
+    this.bqClient = new BigqueryClient("CombinePerKeyExamplesIT");
+    this.bqClient.createNewDataset(this.projectId, this.outputDatasetId, defaultExpiration);
```
Comment (kileys) [inline examples/java/src/test/java/org/apache/beam/examples/cookbook/CombinePerKeyExamplesIT.java:61]: Is there a benefit to creating/deleting the datasets? It looks like some integration tests reuse the same dataset with a table expiration

File: runners/spark/spark_runner.gradle [hunk 3]
```diff
@@ -421,10 +421,11 @@ task examplesIntegrationTest(type: Test) {
   systemProperty "spark.ui.showConsoleProgress", "false"
 
   include '**/*IT.class'
+  exclude '**/TriggerExampleIT.class' // OOM Exception
```
[the remark this answers is not in the document]
Comment (benWize) [reply runners/spark/spark_runner.gradle:424]: No, I will create one and link here

File: sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/testing/BigqueryClient.java [hunk 4]
```diff
@@ -381,6 +381,13 @@ private QueryResponse queryWithRetries(
   /** Creates a new dataset. */
   public void createNewDataset(String projectId, String datasetId)
       throws IOException, InterruptedException {
+    createNewDataset(projectId, datasetId, null);
+  }
+
+  /** Creates a new dataset with defaultTableExpiration. */
+  public void createNewDataset(
+      String projectId, String datasetId, @Nullable Long defaultTableExpiration)
```
[the remark this answers is not in the document]
Comment (benWize) [reply sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/testing/BigqueryClient.java:389]: Yes, I will do

File: examples/java/src/test/java/org/apache/beam/examples/cookbook/CombinePerKeyExamplesIT.java (hunk 2 above)
Comment (benWize) [reply examples/java/src/test/java/org/apache/beam/examples/cookbook/CombinePerKeyExamplesIT.java:61]: I have seen both scenarios, for example, this existing test 

https://github.com/apache/beam/blob/master/examples/java/src/test/java/org/apache/beam/examples/complete/TrafficMaxLaneFlowIT.java#L63

I think creating and deleting a dataset inside the test could help to avoid side effects because of leftover data, but I don't have a strong preference.

**Comment by benWize** [comment]
Run Java Examples_Flink

**Comment by benWize** [comment]
Run Java Examples_Spark

**Comment by benWize** [comment]
Run Java Examples_Direct

**Comment by benWize** [comment]
Run Java Examples_Direct

**Comment by benWize** [comment]
Run Java Examples on Dataflow Runner V2

**Comment by benWize** [comment]
Last test results for each runner:

- [Direct](https://ci-beam.apache.org/job/beam_PostCommit_Java_Examples_Direct/7/)
- [Spark](https://ci-beam.apache.org/job/beam_PostCommit_Java_Examples_Spark/5/)
- [Flink](https://ci-beam.apache.org/job/beam_PostCommit_Java_Examples_Flink/5/)
- [Dataflow](https://ci-beam.apache.org/job/beam_PostCommit_Java_Examples_Dataflow_V2_PR/57/)

File: runners/spark/spark_runner.gradle (hunk 3 above)
[the remark this answers is not in the document]
Comment (benWize) [reply runners/spark/spark_runner.gradle]: Filled https://issues.apache.org/jira/browse/BEAM-14019 to track these

File: sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/testing/BigqueryClient.java (hunk 4 above)
[the remark this answers is not in the document]
Comment (benWize) [reply sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/testing/BigqueryClient.java:389]: Done!

File: sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/testing/BigqueryClient.java [hunk 5]
```diff
@@ -381,6 +381,13 @@ private QueryResponse queryWithRetries(
   /** Creates a new dataset. */
   public void createNewDataset(String projectId, String datasetId)
       throws IOException, InterruptedException {
+    createNewDataset(projectId, datasetId, null);
+  }
+
+  /** Creates a new dataset with defaultTableExpiration in ms. */
+  public void createNewDataset(
+      String projectId, String datasetId, @Nullable Long defaultTableExpiration)
```
Comment (kileys) [inline sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/testing/BigqueryClient.java:389]:
```suggestion
      String projectId, String datasetId, @Nullable Long defaultTableExpirationMs)
```

File: examples/java/src/main/java/org/apache/beam/examples/complete/AutoComplete.java [hunk 6]
```diff
@@ -467,7 +467,6 @@ public void processElement(ProcessContext c) {
 
   public static void runAutocompletePipeline(Options options) throws IOException {
 
-    options.setBigQuerySchema(FormatForBigquery.getSchema());
```
Comment (kileys) [inline examples/java/src/main/java/org/apache/beam/examples/complete/AutoComplete.java:470]: Why are these being removed?

File: runners/flink/flink_runner.gradle [hunk 7]
```diff
@@ -285,8 +285,6 @@ createJavaExamplesArchetypeValidationTask(type: 'Quickstart', runner: 'FlinkLoca
 
 task examplesIntegrationTest(type: Test) {
   group = "Verification"
-  // Disable gradle cache
-  outputs.upToDateWhen { false }
```
Comment (kileys) [inline runners/flink/flink_runner.gradle:289]: Why is this removed?

File: examples/java/src/test/java/org/apache/beam/examples/cookbook/TriggerExampleIT.java [hunk 8]
```diff
@@ -1,100 +0,0 @@
-/*
```
Comment (kileys) [inline examples/java/src/test/java/org/apache/beam/examples/cookbook/TriggerExampleIT.java]: Something wrong with this test?

File: examples/java/src/main/java/org/apache/beam/examples/complete/AutoComplete.java (hunk 6 above)
Comment (benWize) [reply examples/java/src/main/java/org/apache/beam/examples/complete/AutoComplete.java:470]: The property bigQuerySchema fails to serialize and deserialize for portable runners as part of PipelineOptions. 
Stacktrace snippet
`Failed to serialize PipelineOptions
java.lang.IllegalArgumentException: Failed to serialize PipelineOptions
	at org.apache.beam.runners.core.construction.SerializablePipelineOptions.serializeToJson(SerializablePipelineOptions.java:70)`
`Caused by: com.fasterxml.jackson.databind.JsonMappingException: Unexpected IOException (of type java.io.IOException): Failed to serialize and deserialize property 'bigQuerySchema' with value 'GenericData{classInfo=[fields], `

Comment (benWize) [reply examples/java/src/main/java/org/apache/beam/examples/complete/AutoComplete.java:470]: For these examples, the schema is added directly to the transform https://github.com/apache/beam/blob/46155767ecf0694d86257e9c3a8b5fd5bbc0afbb/examples/java/src/main/java/org/apache/beam/examples/complete/AutoComplete.java#L515 using `FormatForBigquery.getSchema()`, not sure why was added to options if it wasn't used from there.

File: examples/java/src/test/java/org/apache/beam/examples/cookbook/TriggerExampleIT.java (hunk 8 above)
Comment (benWize) [reply examples/java/src/test/java/org/apache/beam/examples/cookbook/TriggerExampleIT.java]: The test for TriggerExample is not running for either runner (Flink, Spark, Direct, Dataflow), I removed it from here because I don't see a benefit of adding a non-working IT disabled test. I think further work on this example can be tracked in a different issue, like the created for other disabled examples https://issues.apache.org/jira/browse/BEAM-14019.

File: sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/testing/BigqueryClient.java (hunk 5 above)
Comment (benWize) [reply sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/testing/BigqueryClient.java]: Done!

File: runners/flink/flink_runner.gradle (hunk 7 above)
Comment (benWize) [reply runners/flink/flink_runner.gradle]: I've used some of the configs to launch tests using FlinkRunner from the ValidatesRunner task, I'm not sure if for this case is it worth it to reuse previous outputs from the cache or not. I lean more towards the second.

**Comment by benWize** [comment]
Run Java_Examples_Dataflow_Java11 PreCommit

**Comment by kileys** [comment]
LGTM

**Comment by kileys** [comment]
Run Java PreCommit

**Comment by benWize** [comment]
Run Java PreCommit

**Comment by benWize** [comment]
Run Java PreCommit

**Comment by benWize** [comment]
Thank you both!
